vayesta/ewf/fragment.py

```diff
@@ -278,6 +266,8 @@ def kernel(self, solver=None, init_guess=None, eris=None):
         self._results = results = self.Results(fid=self.id, n_active=cluster.norb_active,
                 converged=cluster_solver.converged, wf=cluster_solver.wf, pwf=pwf)
 
+        eris = cluster_solver.hamil.get_eris_bare()
```

Comment:
Does this recalculate the ERIs?

Can we use the convention: `eris`  and `eris_screened` rather than `eris_bare` and `eris`

Reply:
> Does this recalculate the ERIs?

The ERI's are currently cached within the hamiltonian if calculated (unused configuration option with default True), so won't necessarily be recalculated, but nonetheless this is suboptimal if they haven't previously been calculated. There are some ways to get around this that might be interesting- will add to conversation below.

>Can we use the convention: eris and eris_screened rather than eris_bare and eris

I don't have a hugely strong preference in this context, but thinking about it when we have two possible eris ensuring it's totally clear which is being used might be useful- could we do `eris_bare` and `eris_screened`?